pkg/testutils/integration_test.go

```diff
@@ -0,0 +1,103 @@
+package testutils_test
```

Comment:
This is our entire pipeline integration test, with several assertions embedded therein. The intent is to ensure that adx-mon as installed into a Kubernetes cluster is capable of scraping logs and metrics and ingesting them into Kusto. In addition, we ensure Kusto Views are correct installed and returning the desired schemas.